docs/flink-connector/flink-connector.md

```diff
@@ -35,19 +35,30 @@ This capability allows users to perform federation queries, accessing data from
 | table.catalog-store.kind                         | string | generic_in_memory | The Catalog Store name, it should set to `gravitino`.                | Yes      | 0.6.0-incubating |
 | table.catalog-store.gravitino.gravitino.metalake | string | (none)            | The metalake name that flink connector used to request to Gravitino. | Yes      | 0.6.0-incubating |
 | table.catalog-store.gravitino.gravitino.uri      | string | (none)            | The uri of Gravitino server address.                                 | Yes      | 0.6.0-incubating |
+| table.catalog-store.gravitino.gravitino.client.  | string | (none)            | The configuration key prefix for the Gravitino client config.        | No       | 1.0.0            |
+
+To configure the Gravitino client, use properties prefixed with `table.catalog-store.gravitino.gravitino.client.`. These properties undergo automatic transformation: the prefix is replaced with `gravitino.client.` and passed to the `GravitinoClient`.
+
+**Example:** Setting `table.catalog-store.gravitino.gravitino.client.socketTimeoutMs` is equivalent to setting `gravitino.client.socketTimeoutMs` for the `GravitinoClient`.
```

Comment:
From this example, I only refer that the prefix `table.catalog-store.gravitino` will be trimmed or replaced with an empty string. There seems to be a conflict between this and the phrase "GravitinoClient" and "the prefix is replaced with "gravitino.client."

Reply:
I prefer the sentence like "keys with prefix `table.catalog-store.gravitino` will be trimmed", sentence like replacing "`table.catalog-store.gravitino.gravitino.client.`" with "gravitino.client." seems to be tedious.